src/mono/System.Private.CoreLib/src/System/Reflection/ModifiedType.Mono.cs

```diff
@@ -5,16 +5,151 @@ namespace System.Reflection
 {
     internal partial class ModifiedType
     {
+        /// <summary>
```

Comment:
:+1: This whole file looks much clearer now, thanks!

Reply:
Yes and glad the template worked for mono.